zerver/context_processors.py

```diff
@@ -85,6 +85,9 @@ def zulip_default_context(request):
     if settings.ZILENCER_ENABLED:
         apps_page_url = '/apps/'
 
+    path_parts = [chunk for chunk in request.path.split('/') if chunk != '']  # type: List[str]
+    page_is_help_center = 'help' in path_parts
```

Comment:
This check is super fragile.  I think a better plan would be to have this be set it in the `HelpView.as_view`, since then it only appears in the place that needs it.  Here's the diff that I'm merging that does this.

```
diff --git a/zerver/context_processors.py b/zerver/context_processors.py
index c011a8e..f2163c3 100644
--- a/zerver/context_processors.py
+++ b/zerver/context_processors.py
@@ -85,9 +85,6 @@ def zulip_default_context(request):
     if settings.ZILENCER_ENABLED:
         apps_page_url = '/apps/'
 
-    path_parts = [chunk for chunk in request.path.split('/') if chunk != '']  # type: List[str]
-    page_is_help_center = 'help' in path_parts
-
     user_is_authenticated = False
     if hasattr(request, 'user') and hasattr(request.user, 'is_authenticated'):
         user_is_authenticated = request.user.is_authenticated.value
@@ -128,7 +125,6 @@ def zulip_default_context(request):
         'password_min_length': settings.PASSWORD_MIN_LENGTH,
         'password_min_quality': settings.PASSWORD_MIN_ZXCVBN_QUALITY,
         'zulip_version': ZULIP_VERSION,
-        'page_is_help_center': page_is_help_center,
         'user_is_authenticated': user_is_authenticated,
     }
 
diff --git a/zerver/views/integrations.py b/zerver/views/integrations.py
index dcfc684..2f3b397 100644
--- a/zerver/views/integrations.py
+++ b/zerver/views/integrations.py
@@ -73,6 +73,7 @@ class HelpView(ApiURLView):
 
         # For disabling the "Back to home" on the homepage
         context["not_index_page"] = not path.endswith("/index.md")
+        context["page_is_help_center"] = True
         return context
 
     def get(self, request, article=""):
```